libs/velo-external-db-core/src/web/wix_data_facade.ts

```diff
@@ -4,11 +4,14 @@ import axios from 'axios'
 import { decodeBase64 } from '../utils/base64_utils'
 
 type PublicKeyResponse = {
-    publicKey: string;
+    publicKeys: {
```

Comment:
Maybe create a new type -`PublicKey` and use it instead of `{ [key: string]: string }`.
It would be clearer and more accurate to know what the functions parameters and responses are.